### PR TITLE
Add fips instructions for helm chart

### PR DESCRIPTION
**Pull request by WilliamLoy**
This PR provides the values needed to deploy a Teleport cluster in fips mode using Helm.

### CODE REVIEW

**Review by hugoShaka — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -152,6 +152,17 @@ Do not run debug container images in production environments.
 Heavy container images will continue to be published for Teleport 13 and 14
 throughout the remainder of these releases' lifecycle.
 
+##### Helm cluster chart FIPS mode changes
+
+The teleport-cluster chart no longer uses versionOverride and extraArgs to set FIPS mode. 
+
+Instead, the Values file should set the following:
+```
```
Comment (ptgott) [inline CHANGELOG.md:160]:
```suggestion
Instead, you should use the following values file configuration:
```
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -231,6 +242,17 @@ The operator now joins using a Kubernetes ServiceAccount token. To validate the
 token, the Teleport Auth Service must have access to the `TokenReview` API.
 The chart configures this for you since v12, unless you disabled `rbac` creation.
 
+##### Helm cluster chart FIPS mode changes
+
+The teleport-cluster chart no longer uses versionOverride and extraArgs to set FIPS mode. 
+
+Instead, the Values file should set the following:
+```
```
Comment (ptgott) [inline CHANGELOG.md:250]:
````suggestion
Instead, you should use the following values file configuration:

```
````

Grammar suggestion

File: docs/pages/access-controls/compliance-frameworks/fedramp.mdx [hunk 3]
```diff
@@ -162,6 +162,20 @@ When using `teleport start --fips`, Teleport will start in FIPS mode.
...
+Set the following values in your cluster-values.yaml configuration:
+
+```
+enterpriseImage: public.ecr.aws/gravitational/teleport-ent-fips-distroless
+localAuth: false
+
+```
+
+As of Teleport version 15, versionOverride and extraArgs no longer need to be set
+in the Values configuration file to enable FIPS mode.
```
Comment (ptgott) [inline docs/pages/access-controls/compliance-frameworks/fedramp.mdx:177]:
```suggestion
in the values file to enable FIPS mode.
```
"values file" isn't a proper noun

**Review by ptgott — APPROVED** [review]
Approved with grammar/style suggestions

**Comment by WilliamLoy** [comment]
@ptgott changes approved

File: CHANGELOG.md [hunk 4]
```diff
@@ -152,6 +152,16 @@ Do not run debug container images in production environments.
...
 
+##### Helm cluster chart FIPS mode changes
+
+The teleport-cluster chart no longer uses versionOverride and extraArgs to set FIPS mode. 
+
+Instead, you should use the following values file configuration:
+enterpriseImage: public.ecr.aws/gravitational/teleport-ent-fips-distroless
+localAuth: false
+
+```
```
Comment (reedloden) [inline CHANGELOG.md:163]: Missing this before line 160

**Pull request merged**